webapp/src/components/PublishedData.vue

```diff
@@ -98,12 +98,12 @@ export default defineComponent({
       return urlParts[urlParts.length - 1];
     };
 
-    // Shows the publish time in a more readable format
+    // Shows the UTC publish time in a more readable format
     const formatTime = (timestamp) => {
       const year = timestamp.getFullYear();
       const month = String(timestamp.getMonth() + 1).padStart(2, '0');
       const day = String(timestamp.getDate()).padStart(2, '0');
-      const hours = String(timestamp.getHours()).padStart(2, '0');
+      const hours = String(timestamp.getUTCHours()).padStart(2, '0');
       const minutes = String(timestamp.getMinutes()).padStart(2, '0');
       return `${year}/${month}/${day} ${hours}:${minutes}`;
```

Comment:
add UTC at the end of the string so it would display as:
`2023-09-22 12:26 UTC`
You need to convert all attributes not just minutes to UTC so use getUTCDate, getUTCMinutes etc...